packages/client/src/me.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import fetch from "node-fetch";
+
+export default function Me() {
+  const [me, setMe] = React.useState(null);
+  React.useEffect(() => {
+    fetch("/api/me")
+      .then((data) => data.json())
+      .then((data) => {
+        if (!data.user) return;
+
+        setMe(data.user);
+      });
+  }, []);
+
+  console.log({ me });
+  return <label>{me?.username || "not logged in"}</label>;
```

Comment:
How does `me?.username || "not logged in"` work?

Reply:
see (Optional Chaining)[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining]